docker-compose.yml

```
@@ -48,7 +48,7 @@ services:
     networks:
       - hc-network
     ports:
-      - 80:80
+      - 8001:8001
```

```suggestion
      - 8001:80
```
Good catch!

I think the maintenance screen is just a nginx running on port 80 displaying static html. So you would have to forward port 8001 on host to port 80 in the container.

---

Yeah, I was wondering why I can't run anything before.